client/src/Components/Resuable Components/Header/LoggedInNavigationBar.js

```diff
@@ -25,6 +27,8 @@ class LoggedInNavigationBar extends Component {
       newUrl = "/writeAboutABook";
     } else if (name == "In A Nutshell") {
       newUrl = "/";
+    } else if (name == "N") {
```

Comment:
what does N mean?

Reply:
It's the reduced form of "In a Nutshell" when the screen gets to a certain size. You can try it with the Medium website

Reply:
👍